### PR TITLE
Augment FAQ for Threshold Network merger

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [X] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
Add to the existing Threshold Network merger section of the FAQ.

### CODE REVIEW

**Review by piotr-roslaniec — APPROVED** [review]
LGTM :tada:

File: docs/source/support/faq.rst [hunk 1]
```diff
@@ -11,44 +11,33 @@ The answers in the FAQ regularly reference the network characters "**Alice**," "
...
 Q: How is PRE different from more traditional public key infrastructure (PKI)?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 For an in-depth analysis of this topic, see our blog post entitled "`Three things that NuCypher's "Ursula" can do that vanilla public-key cryptography can't <https://blog.nucypher.com/why-use-nucyphers-ursula-instead-of-traditional-public-key-cryptography/>`_".
 
+Q: If a dApp utilizes the PRE Application, is there an ETH gas fee associated with each re-encryption of encrypted data?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is a gas fee associated with granting and revoking policies. However, there is no gas fee per access.
+Previously policy management was done on L1 but the gas fees were high. Instead, policy management will now
```
Comment (KPrasch) [inline docs/source/support/faq.rst:38]: An attempt to remove the temporal language and state the facts:

Previously policies were published on ethereum mainnet.  As of nucypher 6.0 Policies are published to the polygon sidechain by default for lower gas fees.

File: docs/source/support/faq.rst [hunk 2]
```diff
@@ -96,6 +86,16 @@ Q: How do I integrate nucypher if my application is not written in Python?
 
 Nucypher also provides web integration. See :ref:`web-development`.
 
+.. _faq-testnet:
+
+Q: Are there testnets that I can use?
```
Comment (KPrasch) [inline docs/source/support/faq.rst:91]: This is subject to near-term change as we stabilize the new release across testnet nodes.

File: docs/source/support/faq.rst [hunk 3]
```diff
@@ -11,44 +11,33 @@ The answers in the FAQ regularly reference the network characters "**Alice**," "
...
 
 Q: How is PRE different from more traditional public key infrastructure (PKI)?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 For an in-depth analysis of this topic, see our blog post entitled "`Three things that NuCypher's "Ursula" can do that vanilla public-key cryptography can't <https://blog.nucypher.com/why-use-nucyphers-ursula-instead-of-traditional-public-key-cryptography/>`_".
 
+Q: If a dApp utilizes the PRE Application, is there an ETH gas fee associated with each re-encryption of encrypted data?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is a gas fee associated with granting and revoking policies. However, there is no gas fee per access.
```
[the remark this answers is not in the document]
Comment (derekpierre) [reply docs/source/support/faq.rst]: ✔️

File: docs/source/support/faq.rst (hunk 1 above)
Comment (derekpierre) [reply docs/source/support/faq.rst]: ✔️

File: docs/source/support/faq.rst (hunk 2 above)
Comment (derekpierre) [reply docs/source/support/faq.rst]: ✔️

**Review by KPrasch — APPROVED** [review]

**Pull request merged**